app/views/pages/GUIDE.markdown

```diff
@@ -26,67 +26,64 @@ Please add yourself to this list (link is optional) if you've contributed to thi
 - [Harshdip Singh Deogan](http://hsdeogan.com)  
 - [Jose Nicolas Alba Aguado](http://josealba.org)
 - [Brandon Lucier](https://blucier.com/)  
+- Steven Shlimoon
```

Comment:
You can add your GitHub profile link here if you don't want to link anything else